book/website/project-design/project-management-methodologies/overview.md

```diff
@@ -0,0 +1,30 @@
+# Project Management Methodologies/Frameworks Overview
+
```

Comment:
We should add a brief description here of _what_ a project management framework is here, and how researchers can benefit from applying one

Reply:
Resolving as I added this in a PR instead